opensearch-operator/opensearch-gateway/services/http_request.go

```diff
@@ -68,9 +68,9 @@ func doHTTPHead(ctx context.Context, client *opensearch.Client, path strings.Bui
 	return &opensearchapi.Response{StatusCode: res.StatusCode, Body: res.Body, Header: res.Header}, nil
 }
 
-// doHTTPPut performs a HTTP PUT request
-func doHTTPPut(ctx context.Context, client *opensearch.Client, path strings.Builder, body io.Reader) (*opensearchapi.Response, error) {
```

Comment:
The `doHTTPPut` is still being used https://github.com/search?q=repo%3Aopensearch-project%2Fopensearch-k8s-operator%20doHTTPPut&type=code right?